admin/view-translations-media.php

```diff
@@ -4,30 +4,33 @@
  * Needs WP 3.5+
  *
  * @package Polylang
+ *
+ * @var PLL_Admin_Classic_Editor $this    PLL_Admin_Classic_Editor object.
+ * @var PLL_Language             $lang    The media language. Default language if no language assigned yet.
+ * @var int                      $post_ID The media Id.
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly
-}
+defined( 'ABSPATH' ) || exit;
 ?>
 <p><strong><?php esc_html_e( 'Translations', 'polylang' ); ?></strong></p>
 <table>
 	<?php
 	foreach ( $this->model->get_languages_list() as $language ) {
-		if ( $language->term_id == $lang->term_id ) {
+		if ( $language->term_id === $lang->term_id ) {
 			continue;
 		}
 		?>
 		<tr>
 			<td class = "pll-media-language-column"><span class = "pll-translation-flag"><?php echo $language->flag; // phpcs:ignore WordPress.Security.EscapeOutput ?></span><?php echo esc_html( $language->name ); ?></td>
 			<td class = "pll-media-edit-column">
 				<?php
-				if ( ( $translation_id = $this->model->post->get_translation( $post_ID, $language ) ) && $translation_id !== $post_ID ) {
+				$translation_id = $this->model->post->get_translation( $post_ID, $language );
+				if ( ! empty( $translation_id ) && $translation_id !== $post_ID ) {
 					// The translation exists
 					printf(
 						'<input type="hidden" name="media_tr_lang[%s]" value="%d" />',
 						esc_attr( $language->slug ),
-						esc_attr( $translation_id )
+						esc_attr( (string) $translation_id )
```

Comment:
To be consistent with `%d`.
```suggestion
						(int) $translation_id
```